File: atividade09.c
#include <stdio.h>


int codigo,votos_candidato1,votos_candidato2,votos_branco,votos_nulo;

void Votos(void);

int main()
{

int tipo = 0;
printf("==========2º TURNO ELEIÇÕES 2024=========\n");

    while(tipo == 0)
    {
        printf("\nCandidatos a Prefeitura de Manaus/AM\n");
        printf("[0]  - Branco\n");
        printf("[1]  - Nulo\n");
        printf("[22] - Alberto Neto\n");
        printf("[70] - David Almeida\n\n");
        printf("Digite o código para votar: ");
        scanf("%d", &codigo);

        Votos();

        printf("\============================\n");

        printf("Deseja continuar ?\n[0] - Sim\n[1] - Não\nResposta: ");
        scanf("%d", &tipo);
    }

    printf("Total de Votos para cada Candidato\n");
    printf("Total de votos Alberto Neto: %d\n",votos_candidato1);
    printf("Total de votos David Almeida: %d\n",votos_candidato2);
    printf("Total de Votos nulos: %d\n",votos_nulo);
    printf("Total de Votos em Branco: %d\n",votos_branco);
return 0;
}

void Votos()
{
    switch (codigo)
        {
         case 0:
             votos_branco++;
             printf("\nVoto registrado com sucesso\n");
             break;
         case 1:
             votos_nulo++;
             printf("\nVoto registrado com sucesso\n");
             break;
         case 22:
             votos_candidato1++;
            printf("\nVoto registrado com sucesso\n");
             break;
         case 70:
             votos_candidato2++;
             printf("\nVoto registrado com sucesso\n");
             break;
         default:
             printf("Voto Inválido");
        }
}

File: 2024/atividade_py/atividade09.py
def Votos(codigo):

    global votos_branco,votos_nulo,votos_candidato1,votos_candidato2;

    match codigo:
        case 0:
            votos_branco+= 1;
            print("Voto registrado com sucesso");
        case 1:
            votos_nulo+= 1;
            print("Voto registrado com sucesso");
        case 22:
            votos_candidato1+= 1;
            print("Voto registrado com sucesso");
        case 13:
            votos_candidato2+=2;
            print("Voto registrado com sucesso");
        case _:
            print("Voto Inválido");
    
    
votos_branco = votos_nulo = votos_candidato1 =votos_candidato2 = 0;

tipo = 0;

while tipo == 0:
    print("\nCandidatos a Presidência 2026");
    print("[0]  - Branco");
    print("[1]  - Nulo");
    print("[22] - Bolsonaro");
    print("[13] - Lula\n");
    print("Digite o código para votar: ",end="");
    codigo = int(input());

    Votos(codigo);
    print("============================");
    print("Deseja continuar ?\n[0] - Sim\n[1] - Não\nResposta: ", end="");
    tipo = int(input());


print("Total de Votos para cada Candidato");
print(f"Total de votos Bolsonaro: {votos_candidato1}");
print(f"Total de votos Lula: {votos_candidato2}");
print(f"Total de Votos nulos: {votos_branco}");
print(f"Total de Votos em Branco: {votos_nulo}");

